mininterface/tk_interface/tk_window.py

```diff
@@ -31,6 +30,8 @@ def __init__(self, interface: "TkInterface"):
         self.interface = interface
         self.title(interface.title)
         self.bind('<Escape>', lambda _: self._ok(Cancelled))
+        self.bind('<Control-h>', self._show_help)  # Help with Ctrl+H
+        self.bind('<Control-t>', self._toggle_secrets)
```

Comment:
this imposes a global shortcut. Take look at date_entry – there, the shortcut works only if we are in the field.

That way, it toogles all the secrets in the form, not a single one.

```python
@dataclass
class Credentials:
    password: Annotated[str, SecretTag()]
    api_key: Annotated[str, SecretTag(show_toggle=True)]


m = run(interface="gui")
out = m.form({
    "Password": SecretTag(""),
    "API Key": SecretTag("", show_toggle=True)  # with eye icon
})
print("51: out =", out)
```